themes-default/slim/views/layouts/main.mako

```diff
@@ -61,7 +61,7 @@
 
             <app-header></app-header>
             % if submenu:
-            <%include file="/partials/submenu.mako"/>
+            <submenu></submenu>
```

Comment:
All non page/root components should use a 2 section name like `sub-menu`, `app-header`, etc.